tests/object_storage/test_object_manager.cpp

```diff
@@ -13,22 +15,29 @@ TEST(ObjectManagerTestSuite, TestSetObject) {
     EXPECT_EQ(objectManager.size(), 0);
     EXPECT_EQ(objectManager.sizeUnique(), 0);
 
-    objectManager.setObject(objectID1, std::move(std::vector(payload)));
+    {
+        scaler::object_storage::ObjectPayload tmp = payload;
+        objectManager.setObject(objectID1, std::move(tmp));
+    }
 
     EXPECT_TRUE(objectManager.hasObject(objectID1));
     EXPECT_EQ(objectManager.size(), 1);
     EXPECT_EQ(objectManager.sizeUnique(), 1);
 
     scaler::object_storage::ObjectID objectID2 {3, 2, 1, 0};
 
-    objectManager.setObject(objectID2, std::move(std::vector(payload)));
+    {
+        scaler::object_storage::ObjectPayload tmp = payload;
+        objectManager.setObject(objectID2, std::move(tmp));
```

Comment:
Might be better to call the copy constructor directly

```suggestion
        objectManager.setObject(objectID2, ObjectPayload(payload));
```